### PR TITLE
Fabo/dont query proposer from past chains

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: api/lib/source/cosmosV0-source.js [hunk 1]
```diff
@@ -72,8 +72,8 @@ class CosmosV0API extends RESTDataSource {
   // querying data from the cosmos REST API
   // is overwritten in cosmos v2 to extract from a differnt result format
   // some endpoints /blocks and /txs have a different response format so they use this.get directly
-  async query(url, noRetry) {
-    return this.getRetry(url, noRetry ? 3 : 0)
+  async query(url) {
+    return this.getRetry(url)
```
Comment (Bitcoinera) [inline api/lib/source/cosmosV0-source.js:76]: Should change this also in cosmosV2-source. Will do it

File: api/lib/source/cosmosV0-source.js [hunk 2]
```diff
@@ -339,32 +338,54 @@ class CosmosV0API extends RESTDataSource {
     }
   }
 
-  async getAllProposals(validators) {
-    const response = await this.query('gov/proposals')
-    const { bonded_tokens: totalBondedTokens } = await this.query(
-      '/staking/pool'
+  // we can't query the proposer of blocks from past chains
```
Comment (Bitcoinera) [inline api/lib/source/cosmosV0-source.js:341]: great discovery!

File: api/lib/source/cosmosV0-source.js [hunk 3]
```diff
@@ -121,8 +121,7 @@ class CosmosV0API extends RESTDataSource {
         throw Error()
       }
       const response = await this.query(
-        `slashing/validators/${validatorConsensusPubKey}/signing_info`,
-        { cacheOptions: { ttl: 60 } }
```
Comment (Bitcoinera) [inline api/lib/source/cosmosV0-source.js:125]: what was this cacheOption about? :eyes: not needed?

File: api/lib/source/cosmosV2-source.js [hunk 4]
```diff
@@ -6,8 +6,8 @@ class CosmosV2API extends CosmosV0API {
     this.reducers = require('../reducers/cosmosV2-reducers')
   }
 
-  async query(url, noRetry, resultSelector = 'result') {
-    const response = await this.getRetry(url, noRetry ? 3 : 0)
+  async query(url, resultSelector = 'result') {
+    const response = await this.getRetry(url)
```
Comment (Bitcoinera) [inline api/lib/source/cosmosV2-source.js:10]: ah cool! You didn't forget about it. Amazing :woman_dancing:

**Review by Bitcoinera — APPROVED** [review]

File: api/lib/source/cosmosV0-source.js (hunk 3 above)
Comment (faboweb) [reply api/lib/source/cosmosV0-source.js:125]: never used

**Pull request merged**